src/features/helm-charts/add-helm-repository-from-list-in-preferences.test.ts

```diff
@@ -129,9 +128,13 @@ describe("add helm repository from list in preferences", () => {
 
           describe("when adding rejects", () => {
             beforeEach(async () => {
-              await execFileMock.reject(
-                "Some error",
-              );
+              await execFileMock.resolve({
+                callWasSuccessful: false,
+                error: {
```

Comment:
Yes I agree but sometimes it is useful to get the error object itself, and sometimes it is useful to get the stderr output.

Reply:
We could have that, and less funk with eg. `error.errorObject` or `error.rejection`? 

Also it feels like something called `stderr` doesn't belong in this layer of abstraction. Should it be `error.errorMessage`?

Reply:
Why not always pass object of type `Error`, and the caller can access `error.message` as normal. If stderr is required it could be a property of error object (and we could extend the `Error` class to have type safety).

Reply:
We are passing the `Error` object too. We will certain work to make our error handling story better.

Reply:
I know, I was just making a suggestion to fix the "error.error seems odd" and also simplifying things. But I understand this could be a separate change.